### PR TITLE
add pubs up to f24

### PR DESCRIPTION
**Pull request by a-wagner**

### CODE REVIEW

File: publications.rkt [hunk 1]
```diff
@@ -67,6 +67,51 @@
 
 @(define publications
    (list
+    (publication "Realistic Realizability: Specifying ABIs You Can Count On"
+                 "Andrew Wagner, Zachary Eisbach, and Amal Ahmed"
+                 "Object-Oriented Programming Systems, Languages, and Applications (OOPSLA)"
+                 2024
+                 "https://doi.org/10.1145/3689755")
```
Comment (tybug) [inline publications.rkt:74]: this doi 404's for me 🤔

**Review by tybug — APPROVED** [review]
lgtm otherwise

File: publications.rkt (hunk 1 above)
Comment (ulysses4ever) [reply publications.rkt:74]: It's probably fine: these DOI go into effect closer to the conference, which, in this case of OOPSLA'24, happens in the late October.

Comment (tybug) [reply publications.rkt:74]: ok cool, thanks!

**Pull request merged**